lib/changebase/active_record/helpers.rb

```diff
@@ -0,0 +1,27 @@
+module Changebase::ActiveRecord::Helpers
+  
+  def identity_digest
```

Comment:
Did you want to change this to `calculate_identity_digest` or `changebase_identity_digest` to avoid collision with the identity digest of Events?